ddl/delete_range.go

```diff
@@ -239,12 +239,12 @@ func insertJobIntoDeleteRangeTable(ctx sessionctx.Context, job *model.Job) error
 		tableID := job.TableID
 		// The startKey here is for compatibility with previous versions, old version did not endKey so don't have to deal with.
 		var startKey kv.Key
-		var partitionIDs []int64
-		if err := job.DecodeArgs(startKey, &partitionIDs); err != nil {
+		var physicalIDs []int64
```

Comment:
I suggest using physicalTableID outside the table package. Or it doesn't have a clear meaning.